SPECS/glibmm/glibmm.spec

```diff
@@ -0,0 +1,159 @@
+%global apiver 2.68
+
+Summary:        C++ interface to the glib
+Name:           glibmm
+%define BaseVersion 2.70
+Version:        %{BaseVersion}.0
+Release:        1%{?dist}
+License:        LGPLv2+
+URL:            https://developer.gnome.org/glibmm/stable/
+Group:          Applications/System
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        http://ftp.gnome.org/pub/GNOME/sources/glibmm/%{BaseVersion}/glibmm-%{version}.tar.xz
+BuildRequires:  doxygen
+BuildRequires:  gcc-c++
+BuildRequires:  glib-devel 
+BuildRequires:  glib-schemas
+BuildRequires:  graphviz
+BuildRequires:  libsigc++30
+BuildRequires:  libxslt
+BuildRequires:  m4
+BuildRequires:  meson
+BuildRequires:  mm-common
+BuildRequires:  perl-generators
+BuildRequires:  perl-interpreter
+
+%global with_check 0
+%if %{with_check}
+BuildRequires:  glib-networking
+%endif
+
+Requires:	libsigc++30
+Requires:	glib >= 2.69.1
+Requires:	gobject-introspection >= 1.50.0
+Requires:	perl-XML-Parser
+
+%description
+gtkmm provides a C++ interface to the GTK+ GUI library. gtkmm2 wraps GTK+ 2.
+Highlights include typesafe callbacks, widgets extensible via inheritance and
+a comprehensive set of widget classes that can be freely combined to quickly create complex user interfaces.
+
+%package devel
+Summary: Header files for glibmm
+Group: Applications/System
+Requires: 	%{name} = %{version}
+Requires:	glib-devel
```

Comment:
Since it looks like a new spec added to the `SPECS` folder, could you also run a linter against it for some basic clean-up?